### ISSUE TITLE
Bug: (fill in)access Script Console Jenkins lead to do rce d4science‏

### ISSUE THREAD

**Issue by shadihh9** [eslint/eslint#19267]
### Environment

linux 

### What parser are you using?

Default (Espree)

### What did you do?

<details>
<summary>Configuration</summary>

```
i found bug access Script Console Jenkins lead to do rce jenkins2.eslint.org


<!-- Paste your configuration here -->
```
</details>

```js
<!-- Paste your code here -->
```

### What did you expect to happen?

can attacker access Script Console Jenkins lead to do rce

### What actually happened?

1- go to http://146.48.122.11:8080/_script and see   access Script
Console Jenkins lead to do rce  jenkins2.eslint.org 
  

### Link to Minimal Reproducible Example

https://stackblitz.com/ab222

### Participation

- [ ] I am willing to submit a pull request for this issue.

### Additional comments

<img width="1366" alt="Image" src="https://github.com/user-attachments/assets/d9f003b1-1beb-4eec-bcb3-17a7c9fc6bc6" />

**Comment by mdjermanovic** [comment]
This doesn't seem to be related to our server. In the future, please report issues of this kind through security advisory forms on GitHub.
